<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-6P9V2RS" failures="3" tests="5" name="Test" time="336.198" errors="0" timestamp="2024-06-16T20:28:53 IST">
  <testcase classname="scenario1.HRMtest" name="login" time="92.813"/>
  <testcase classname="scenario1.HRMtest" name="login" time="47.675">
    <failure type="java.lang.AssertionError" message="Login fail expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Login fail expected [true] but found [false]
at scenario1.HRMtest.login(HRMtest.java:40)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- login -->
  <testcase classname="scenario1.HRMtest" name="login" time="32.837">
    <failure type="java.lang.AssertionError" message="Login fail expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Login fail expected [true] but found [false]
at scenario1.HRMtest.login(HRMtest.java:40)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- login -->
  <testcase classname="scenario1.HRMtest" name="login" time="114.426"/>
  <testcase classname="scenario1.HRMtest" name="login" time="44.922">
    <failure type="java.lang.AssertionError" message="Login fail expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Login fail expected [true] but found [false]
at scenario1.HRMtest.login(HRMtest.java:40)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- login -->
</testsuite> <!-- Test -->
